README.md

```diff
@@ -0,0 +1,112 @@
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead, if you have questions about the R&A process.<br/><br/>
+
+* **Step 1) Create your own fork of the kokkos-core-wiki repo by navigating to `https://github.com/kokkos/kokkos-core-wiki`, and clicking "Fork" in the upper right of the github page**.<br/>
+
+* **Step 2) Clone your forked repo, and add it as a remote:**<br/>
+
+```
+git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
+git remote add ajpowelsnl
+git@github.com:ajpowelsnl/kokkos-core-wiki.git
+```
+
+* **Step 3) Add the remote repo as "origin" or "upstream", according to your preference.  You will create pull requests against this repo:**<br/>
+
+```
+git remote add origin git@github.com:kokkos/kokkos-core-wiki.git
+```
+
+* **Step 4) Check your remotes; you should see both your fork (`ajpowelsnl`) and the main project repo (`origin`):**<br/>
+
+```
+[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+origin https://github.com/kokkos/kokkos-core-wiki.git (fetch)
+origin https://github.com/kokkos/kokkos-core-wiki.git (push)
+```
+
+* **Step 5) Create a topic branch in your local fork of the kokkos-core-wiki:**<br/>
+
+```
+git checkout -b demo/setup
+```
+
+* **Step 6) Add the Kokkos project Wiki repo from your fork as a remote (you will push changes to this repo to preview before creating a pull request on `origin git@github.com:kokkos/kokkos-core-wiki.git`):**<br/>
+
+```
+git remote add my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git
+```
+
+* **Step 7) Verify your remotes.  You should now have three different repos: your local, forked kokkos-core-wiki (`ajpowelsnl`), the remote kokkos-core-wiki (`origin`), and the Wiki associated with your forked Kokkos project (`my_wiki`):**<br/>
+
+```
+[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git (fetch)
+my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git (push)
+origin https://github.com/kokkos/kokkos-core-wiki.git (fetch)
+origin https://github.com/kokkos/kokkos-core-wiki.git (push)
+```
+
+* **Step 8) Set up your local topic branch to track origin/main:**<br/>
+
+```
+git checkout main
+git branch --set-upstream-to=origin/main 
+git pull origin/main
+git show
+commit f0e10f59f50f23930a2f4eedfce3eed869344277 (HEAD -> main, origin/main)
+Merge: 7b5063e ec9f444
+Author: Damien L-G <dalg24@gmail.com>
+Date:   Mon Feb 28 21:09:11 2022 -0500
+
+Merge pull request #3 from dalg24/fixup_gh_actions
+
+Fixup path gh workflow
+```
+
+* **Step 9) Update your topic branch, rebasing on origin/main:**<br/>
+
+```
+git checkout demo/setup
+git rebase main
+```
```

Comment:
Again, I don't see much of a reason to bother with the local `main` branch and would just do
```
git fetch origin main
git checkout demo/setup
git rebase origin/main
```
Most importantly,
```
git fetch origin/main
```
should be
```
git fetch origin main
```